samples/experiment.yaml

```diff
@@ -0,0 +1,11 @@
+name: Benchmarking DNA
+runs: 3
+workflow_conf:
+  - benchmarks/example.yml
+  - benchmarks/dna-named.yml
+  - benchmarks/rna-named.yml
+cloud:
+  - iu2
+job_configs:
+  - rules/4x8.yml
+  - rules/8x16.yml
```

Comment:
rule files need sample content

Reply:
And probably more explanation as the format of the files will depend on what chart was used to install Galaxy.  Since GalaxyKubeman installs Galaxy as a dependent chart there is a top level `galaxy:` key that is not needed if Galaxy is installed from the `galaxy-helm` chart.

I will also give some thought to rewriting this section.  For the cloud costs we only update TPV rules, but any values from the Chart can be specified and updated here.